core/templates/dev/head/domain/exploration/EditableExplorationBackendApiServiceSpec.ts

```diff
@@ -19,6 +19,8 @@
 require('domain/exploration/EditableExplorationBackendApiService.ts');
 require('domain/exploration/ReadOnlyExplorationBackendApiService.ts');
 require('services/CsrfTokenService.ts');
+require(
+  'pages/exploration-editor-page/services/user-exploration-permissions.service.ts');
```

Comment:
Move this before the `require('services/CsrfTokenService.ts');`

Reply:
Done